robolectric/src/test/java/org/robolectric/shadows/ShadowBitmapTest.java

```diff
@@ -293,6 +293,39 @@ public void throwsExceptionForZeroHeight() {
     Bitmap.createBitmap(100, 0, Bitmap.Config.ARGB_8888);
   }
 
+  @Test
+  public void shouldGetPixelsFromAnyNonNullableCreatedBitmap() {
+    // todo: Test for each of the create methods that getPixels() is not null for 10x10
+    Bitmap bitmap;
+    int width = 10;
+    int height = 10;
+
+    int[] pixels = new int[width * height];
+    bitmap = Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888);
+    bitmap.getPixels(pixels, 0, width, 0, 0, width, height);
+    assertThat(pixels).isNotNull();
```

Comment:
This test isn't right because pixels will never be null since you are creating it in the test.